index.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Lato:400,700"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Titan+One&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Healthy Food</title>
+  </head>
+  <body>
+    <div class="container-fluid">
+      <div class="row">
+        <nav
+          class="navbar navbar-inverse text-white socialMediaIcons"
+          id="home"
+        >
+          <div class="icon-bar navbar-right">
+            <a href="#" class="facebook"><i class="fa fa-facebook"></i></a>
+            <a href="#" class="twitter"><i class="fa fa-twitter"></i></a>
+            <a href="#">My page</a>
+            <a href="#">Logout</a>
+          </div>
+        </nav>
+      </div>
+      <div class="row">
+        <nav class="navbar" id="navbar">
+          <div class="navbar-header">
+            <a class="navbar-brand" href="#">
+              <img src="./images/img4.png" alt="navbar logo imag" />
+            </a>
+          </div>
+          <ul class="nav navbar-nav navItem text-secondary" id="navMenu">
+            <li class="active"><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#program">Program</a></li>
+            <li><a href="#sports">Sports</a></li>
+            <li><a href="#sponsers">Sponser</a></li>
+            <li><a href="#">Nutrition</a></li>
+          </ul>
+
+          <div class="humgurger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+        </nav>
+      </div>
+      <main class="main1">
+        <div class="row introduction">
+          <p>"Welcome to Healthline Nutrition!"</p>
+          <h1 class="secondHeader">
+            The Definitive Guide<br />
+            to Healthy Eating in Real Life
+          </h1>
```

Comment:
- [ ] For the desktop view make sure that this `introduction` section is well aligned to the `left` with your `logo` in the navigation bar.
Your design 
![Screenshot from 2022-10-25 17-58-31](https://user-images.githubusercontent.com/89228743/197856902-7dc72ae7-8554-4f5e-b79f-c5cdebad039d.png)

The template 
![Screenshot from 2022-10-25 18-49-57](https://user-images.githubusercontent.com/89228743/197857204-cdba0428-764c-4f25-a6cb-d66f80e3627d.png)